rumi/file_rumi/reader.py

```diff
@@ -378,7 +378,9 @@ def parse_history(self):
 
                 add = commit.stats.files[file]["insertions"]
                 delete = commit.stats.files[file]["deletions"]
-                n_lines = commit.stats.files[file]["lines"]
+                # Get total number of lines of files directly from the file
+                with open(self.repo_path / fname, "r+") as f:
+                    n_lines = len(f.readlines())
```

Comment:
You mean like in the case of errors like this `FileNotFoundError: [Errno 2] No such file or directory`? Will n_lines be count as 0?

Reply:
Yes, if I'm interpreting this correctly if the file does not already exist an empty file will be created, which will make `n_lines` equal to 0. This might cause us some issues later down in the code so my suggestion is to raise an Exception in that case.